src/guides/v2.3/ui_comp_guide/concepts/ui_comp_modifier_concept.md

```diff
@@ -103,7 +103,17 @@ where `YourNamespace\YourModule\DataProvider\Modifier\Pool` is a [virtual class]
 
 **Step 3:**
 
-To use your modifier, add a dependency on [`\Magento\Ui\DataProvider\Modifier\PoolInterface`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/DataProvider/Modifier/PoolInterface.php) to your UI component data provider. For illustration see [`\Magento\Catalog\Ui\DataProvider\Product\Form\ProductDataProvider`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Catalog/Ui/DataProvider/Product/Form/ProductDataProvider.php)
+To use your modifier, add a dependency on [`\Magento\Ui\DataProvider\Modifier\PoolInterface`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/DataProvider/Modifier/PoolInterface.php) to your UI component data provider.
+
+Sample dependency injection:
+
+```xml
+<type name="%YourNamespce\YourModule\Ui\DataProvider\YourDataProviderClass%">
+    <arguments>
+        <argument name="pool" xsi:type="object">%YourNamespace\YourModule\DataProvider\Modifier\Pool%</argument>
+    </arguments>
+</type>
+```
 
```

Comment:
```suggestion

For illustration see [`\Magento\Catalog\Ui\DataProvider\Product\Form\ProductDataProvider`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Catalog/Ui/DataProvider/Product/Form/ProductDataProvider.php).
```

I would recommend to keep the reference to 
` \Magento\Catalog\Ui\DataProvider\Product\Form\ProductDataProvider` class as an example. Because it might be unclear what "add a dependency on \Magento\Ui\DataProvider\Modifier\PoolInterface" means.